recipes/thuner/meta.yaml

```
@@ -0,0 +1,86 @@
+{% set name = "thuner" %}
+{% set version = "0.0.7" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b94c0120c930a58059020d2941fb680b3fe87fc70d3a88c7a1fa4fac5dc2f21e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip:
+    - win
```

```suggestion
  skip: True  # [win]
```
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#skipping-builds